### ISSUE TITLE
Problem after installation

### ISSUE THREAD

**Issue by Rain-1023** [AymanLyesri/hyprland-conf#109]
It's possible to install your rice easily?, I tried to install it, but there are no backgrounds or polybar, and at the end of the installation, it showed an error in dbus.

**Comment by Rain-1023** [comment]
IDK what im doing wrong but nothing works, i use super + w and doesnt works and the app launcher neither

**Comment by AymanLyesri** [comment]
run ags in the terminal and post the output

**Comment by AymanLyesri** [comment]
no updates, closing this issue...

**Comment by HinaHyugaHime** [comment]
> IDK what im doing wrong but nothing works, i use super + w and doesnt works and the app launcher neither

It'll be fixed later, I notified him of the issue, he hasn't updated his apps so it's still working on his

**Comment by AymanLyesri** [comment]
There is a new ags update that breaks everything sadly, all the code needs to be rewritten and it would take sometime before its ready, be sure to read the read-me for a temporary workaround

**Comment by AymanLyesri** [comment]
run `update` in the terminal it will install ags V1
then either run `agsv1` or press `SUPER + b`

**Comment by HinaHyugaHime** [comment]
> run `update` in the terminal it will install ags V1 then either run `agsv1` or press `SUPER + b`

Can't fix v2?

**Comment by AymanLyesri** [comment]
the whole AGS code needs to be rewritten, its not a simple tweak, it could take me weeks before migrating all widgets to V2, so annoying actually

**Comment by HinaHyugaHime** [comment]
rip, lmk when done

**Comment by Rain-1023** [comment]
I'm here again regarding the installation failure. I think it's because I don't have /home, If the issue is because of that, I honestly have no idea what to do.

**Comment by AymanLyesri** [comment]
> I'm here again regarding the installation failure. I think it's because I don't have /home, If the issue is because of that, I honestly have no idea what to do.

u should have /home as its standard for most linux distribution, are u sure u running linux?

anyway try creating the directory and let me know
```bash
mkdir /home
```

also run `agsv1` and post the output

**Comment by Rain-1023** [comment]
(com.github.Aylur.ags:3697): Gjs-CRITICAL **: 12:40:08.666: Attempting to call back into JSAPI during the sweeping phase of GC. This is most likely caused by not destroying a Clutter actor or Gtk+ widget with ::destroy signals connected, but can also be caused by using the destroy(), dispose(), or remove() vfuncs. Because it would crash the application, it has been blocked and the JS callback not invoked.
The offending signal was destroy on Ags_Label 0x5c0cac2682d0.

**Comment by Rain-1023** [comment]
had an older version of AGS, I updated it, and now when I run AGSV1, this message appears: 'AGS with busname "ags" is already running.

**Comment by HinaHyugaHime** [comment]
> > I'm here again regarding the installation failure. I think it's because I don't have /home, If the issue is because of that, I honestly have no idea what to do.
> 
> u should have /home as its standard for most linux distribution, are u sure u running linux?
> 
> anyway try creating the directory and let me know
> 
> mkdir /home
> 
> also run `agsv1` and post the output

if he is genuinely missing /home he needs to reinstall his distro because /home/$USER is what holds everything

**Comment by AymanLyesri** [comment]
> if he is genuinely missing /home he needs to reinstall his distro because /home/$USER is what holds everything

yep, better reinstall arch with Hyprland, u will find /home and its users already populated, after that just run the install script

**Comment by Rain-1023** [comment]
I already did it, and it still doesn't work. I'm tired; I think I won't use your rice after all. :(

**Comment by AymanLyesri** [comment]
Sorry to hear that, sometimes u will face obstacles but u need to keep pushing forward, if u got any issues just let me know am happy to help.

**Comment by Rain-1023** [comment]
For some strange reason, it works now ._. Thanks for the advice, I'll keep it in mind. By the way, the best Linux rice I've ever seen.

**Comment by AymanLyesri** [comment]
Glad it worked out for u, if u have any ideas or suggestions let me know.
